k8s/grafana/apptest/tester/tests/basic-suite.yaml

```diff
@@ -1,8 +1,22 @@
 actions:
-- name: dummy test
+- name: grafana listening
   bashTest:
     script: |-
-      exit 0
+      sleep 120
```

Comment:
Why sleep? Please [wait for ready status](https://github.com/GoogleCloudPlatform/click-to-deploy/blob/master/k8s/nginx/apptest/tester/tests/basic-suite.yaml.template#L10).
```
kubectl wait pod/${POD_NAME} \
          --namespace=${NAMESPACE} \
          --for=condition=Ready \
          --timeout=180s
```